common/src/main/kotlin/org/valkyrienskies/mod/common/PlayerUtil.kt

```diff
@@ -13,10 +13,8 @@ object PlayerUtil {
     // Updates player to 'live' in ship space for everything executed in the inside lambda
     // is used for emulating the environment when you interact with a block
     fun <T> transformPlayerTemporarily(player: PlayerEntity, ship: ShipObject?, inside: () -> T): T {
-        val tmpYaw = if (player.world.isClient)
-            player.yaw
-        else
-            player.headYaw
+        val tmpYaw = player.yaw
+        val tmpHeadYaw = player.headYaw
```

Comment:
Thx for fixing this :)

Reply:
yhea did it in here since, it wouldn't be like a full pr worth for that